.github/workflows/website.yml

```
@@ -5,70 +5,140 @@ on:
-
-      - name: Run linters
-        run: npm run website:lint
-
-      - name: Run typecheck
-        run: npm run website:build-paths && npm run typecheck
+  build:
+    name: "🏗️ Build Website"
+    runs-on: ubuntu-latest
+    needs: test
+    outputs:
+      build_uploaded: ${{ steps.upload.outputs.ok }}
```

actionlint (1.7.10)

property "ok" is not defined in object type {artifact-digest: string; artifact-id: string; artifact-url: string}

(expression)